packages/orchestrator/lib/clients/processor.ts

```diff
@@ -41,6 +41,10 @@ export class OrchestratorProcessor {
         }
     }
 
+    public queueSize() {
```

Comment:
I don't love exposing this but it's the easiest way to help the test know if the queue has emptied.